plog-kafka/src/main/java/com/airbnb/plog/kafka/KafkaHandler.java

```diff
@@ -77,6 +86,7 @@ public JsonObject getStats() {
         return new JsonObject()
                 .add("default_topic", defaultTopic)
                 .add("seen_messages", seenMessages.get())
+                .add("propagated_messages", propagatedMessages.get())
```

Comment:
The `propagate` flag value is the same for all messages, to `seen_messages` and `propagated_messages` will always be the same. amirite?
I'd then remove than metric.